### PR TITLE
fix(14534): make the azure vectorsearch compatible with latest stable release 11.4

### PR DESCRIPTION
**Pull request by PascalGiessler**
- **Description:** This fix solves the issue of being compatible with the latest stable version azure search documents 11.4. 
  - **Issue:** 14534
  - **Dependencies:** none
  - **Tag maintainer:** none
  - **Twitter handle:** none

### CODE REVIEW

**Review by baskaryan — COMMENTED** [review]
cc @ruoccofabrizio

File: libs/community/langchain_community/vectorstores/azuresearch.py [hunk 1]
```diff
@@ -431,16 +431,16 @@ def vector_search_with_score(
...
-        from azure.search.documents.models import Vector
+        from azure.search.documents.models import VectorizedQuery
 
         results = self.client.search(
             search_text="",
-            vectors=[
-                Vector(
+            vector_queries=[
+                VectorizedQuery(
                     value=np.array(
```
Comment (baskaryan) [inline libs/community/langchain_community/vectorstores/azuresearch.py:440]: is `value` supposed to be `vector` now?

Comment (PascalGiessler) [reply libs/community/langchain_community/vectorstores/azuresearch.py]: @baskaryan You were right. I missed that to copy it over to my other repo. I've changed it.

**Comment by baskaryan** [comment]
believe this was resolved in #15659, let me know if i'm missing something though!

**Pull request closed without merging**